code/modules/projectiles/guns/sentries.dm

```diff
@@ -199,24 +198,28 @@
 	default_ammo_type = /obj/item/ammo_magazine/sentry/fob_sentry
 	allowed_ammo_types = list(/obj/item/ammo_magazine/sentry/fob_sentry)
 
-/obj/item/storage/box/minisentry
+/obj/item/storage/box/crate/minisentry
 	name = "\improper ST-580 point defense sentry crate"
 	desc = "A large case containing all you need to set up an ST-580 point defense sentry."
-	icon = 'icons/Marine/marine-weapons.dmi'
-	icon_state = "sentry_case"
+	icon_state = "sentry_mini_case"
 	w_class = WEIGHT_CLASS_HUGE
 	storage_slots = 6
 	can_hold = list(
 		/obj/item/weapon/gun/sentry/mini,
 		/obj/item/ammo_magazine/minisentry,
 	)
 
-/obj/item/storage/box/minisentry/Initialize(mapload, ...)
+/obj/item/storage/box/crate/minisentry/Initialize(mapload, ...)
 	. = ..()
 	new /obj/item/weapon/gun/sentry/mini(src)
 	new /obj/item/ammo_magazine/minisentry(src)
 	new /obj/item/ammo_magazine/minisentry(src)
 
+/obj/item/storage/box/crate/minisentry/update_icon_state()
+	icon_state = initial(icon_state)
+	if(!length(contents))
+		icon_state = "empty_case"
```

Comment:
Why does this exist?

Reply:
Forgot to get rid of them. I'll remove them.